lib/decidim/social_crowdfunding/goteo/api.rb

```diff
@@ -5,34 +5,39 @@ module SocialCrowdfunding
     module Goteo
       module Api
         PROJECT_URL = "%{base_url}/projects/%{slug}"
-        PROJECTS_URL = "%{base_url}/projects/?limit=%{limit}&page=%{page}"
 
         class << self
           def base_url
-            "https://api.goteo.org/v1"
+            ENV.fetch("GOTEO_BASE_URL", "https://api.goteo.org/v1")
```

Comment:
That's ok, but i'd use a configuration option for the plugin in the future. Maybe add a Fixme comment

Reply:
Can you share the link with an example? I couln't find a reference for the way we usually set this up.

Reply:
Also a great way to have options for different configuration and explain them. https://github.com/Platoniq/decidim-module-civicrm/blob/main/lib/decidim/civicrm.rb